packages/apiserver/src/migrations/1661339543400-migrationMy.ts

```diff
@@ -0,0 +1,13 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class migrationMy1661339543400 implements MigrationInterface {
+  name = 'migrationMy1661339543400';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`ALTER TABLE "person" DROP COLUMN "location_id"`);
+  }
+
+  public async down(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`ALTER TABLE "person" ADD "location_id" integer`);
+  }
+}
```

Comment:
Are there any reasons not to merge these migrations into one?
Also, could you use more sane name other than `migrationMy` please?